metagraph/tests/graph/all/test_dbg_helpers.hpp

```diff
@@ -93,7 +100,25 @@ typedef ::testing::Types<DBGBitmap,
                          DBGSuccinctBloomFPR<1, 1>,
                          DBGSuccinctBloomFPR<1, 10>,
                          DBGSuccinctBloom<4, 1>,
-                         DBGSuccinctBloom<4, 50>> GraphTypes;
+                         DBGSuccinctBloom<4, 50>,
+                         DBGSuccinctCachedWrapped> GraphTypes;
+
+template <typename Graph>
+class MutableDeBruijnGraphTest : public ::testing::Test { };
+typedef ::testing::Types<DBGBitmap,
+                         DBGHashString,
+                         DBGHashOrdered,
+                         DBGHashFast,
+                         DBGSuccinct,
+                         DBGSuccinctIndexed<1>,
+                         DBGSuccinctIndexed<2>,
+#if ! _PROTEIN_GRAPH
+                         DBGSuccinctIndexed<10>,
+#endif
+                         DBGSuccinctBloomFPR<1, 1>,
+                         DBGSuccinctBloomFPR<1, 10>,
+                         DBGSuccinctBloom<4, 1>,
+                         DBGSuccinctBloom<4, 50>> MutableGraphTypes;
```

Comment:
Why are `DBGSuccinctBloom` and `DBGSuccinctIndexed` are mutable, but not `DBGSuccinctCachedWrapped`?

Reply:
serialize and load are not implemented in `DBGSuccinctCachedWrapped`

Reply:
Why not? Just reset the cache and call the base functions.

Reply:
I had it at some point and you asked me to remove it, but I don't remember if this was for this class or the parent wrapper class

Reply:
it'll make things more complicated because the wrapper class would have to store a non-const shared_ptr

Reply:
Ok, this sounds like a reason.
For serialize/load nothing prevents us to implement those.